pom.xml

```diff
@@ -30,28 +30,28 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>1.642.1</jenkins.version>
```

Comment:
Maybe it's too early to bump to this version. No strong opinion, but it may impact @ikedam's plans with copy artifact